generated-docs/FRP/Event.md

```diff
@@ -54,49 +63,84 @@ IsEvent Event
 never :: forall a. Event a
 ```
 
-#### `filter`
+#### `keepLatest`
 
 ``` purescript
-filter :: forall a. (a -> Boolean) -> Event a -> Event a
+keepLatest :: forall a. Event (Event a) -> Event a
 ```
 
-Create an `Event` which only fires when a predicate holds.
+Flatten a nested `Event`, reporting values only from the most recent
+inner `Event`.
 
 #### `subscribe`
 
 ``` purescript
-subscribe :: forall eff a r. Event a -> (a -> Eff (frp :: FRP | eff) r) -> Eff (frp :: FRP | eff) Unit
+subscribe :: forall eff a r. Event a -> (a -> Eff (frp :: FRP | eff) r) -> Eff (frp :: FRP | eff) (Eff (frp :: FRP | eff) Unit)
 ```
 
 Subscribe to an `Event` by providing a callback.
 
-#### `create`
+`subscribe` returns a canceller function.
+
+
+### Re-exported from Data.Filterable:
+
+#### `Filterable`
 
 ``` purescript
-create :: forall eff a. Eff (frp :: FRP | eff) { event :: Event a, push :: a -> Eff (frp :: FRP | eff) Unit }
+class (Functor f) <= Filterable f  where
+  filterMap :: forall a b. (a -> Maybe b) -> f a -> f b
 ```
 
-Create an event and a function which supplies a value to that event.
+`Filterable` represents data structures which can be _partitioned_/_filtered_.
 
+- `partitionMap` - partition a data structure based on an either predicate.
+- `partition` - partition a data structure based on boolean predicate.
+- `filterMap` - map over a data structure and filter based on a maybe.
+- `filter` - filter a data structure based on a boolean.
+
+Laws:
+- `map f ≡ filterMap (Just <<< f)`
+- `filter ≡ filterMap <<< maybeBool`
+- `filterMap p ≡ filter (isJust <<< p)`
+
+Default implementations are provided by the following functions:
+
+- `partitionDefault`
+- `partitionDefaultFilter`
+- `partitionDefaultFilterMap`
+- `filterDefault`
+- `filterDefaultPartition`
+- `filterDefaultPartitionMap`
+
+##### Instances
+``` purescript
+Filterable Array
+Filterable Maybe
+(Monoid m) => Filterable (Either m)
+Filterable List
+(Ord k) => Filterable (Map k)
+```
 
 ### Re-exported from FRP.Event.Class:
 
 #### `IsEvent`
 
 ``` purescript
-class (Alternative event) <= IsEvent event  where
+class (Alternative event, Filterable event) <= IsEvent event  where
```

Comment:
Aaah... I forgot to do this, didn't I? Sorry about that.

Reply:
No, you didn't, it's just GitHub being weird about merges.